lib/global_id/locator.rb

```diff
@@ -139,6 +126,26 @@ class ActiveRecordFinder
         def locate(gid)
           gid.model_class.find gid.model_id
         end
+
+        def locate_many(gids, options = {})
+          models_and_ids  = gids.collect { |gid| [ gid.model_class, gid.model_id ] }
+          ids_by_model    = models_and_ids.group_by(&:first)
+          loaded_by_model = Hash[ids_by_model.map { |model, ids|
+            [ model, find_records(model, ids.map(&:last), ignore_missing: options[:ignore_missing]).index_by { |record| record.id.to_s } ]
```

Comment:
Right, I just feel like `locate_many` might leave too much to custom locators. But if so that's still an abstraction I can't distill today. So let's leave it to your solution, 👍.